README.md

```diff
@@ -1,101 +1,137 @@
 
-A collection of typescript tools to interact with LooksRare protocol V2 smart contracts 👀💎
+A collection of TypeScript tools to interact with HypercertsExchange smart contracts, enabling users to sell and buy Hypercerts on the marketplace.
 
-## Install
 
-This package has a peer dependency on [etherjs V5](https://docs.ethers.io/v5/).
+## SDK usage
 
-```bash
-yarn add @looksrare/sdk-v2 ethers
-```
+<!-- TODO: Check if links still work on deployment -->
+Read the [guides](./guides) if you need help with the implementation.
 
-```bash
-npm install @looksrare/sdk-v2 ethers --save
-```
+You can also read the detailed [API documentation](./doc).
 
-## Getting Started
+## SDK development
 
-The SDK expose a main class used to perform all the onchain operations:
+### Environment variables
 
-```ts
-import { ChainId } from "@looksrare/sdk-v2";
-const looksrare = new LooksRare(ChainId.MAINNET, provider, signer);
-```
+No environment variables are required to run, build or run tests on the SDK.
 
-The signer is optional if you need access to read only data (:warning: Calls to function that need a signer will throw a `Signer is undefined` exception):
+### Local development
 
-```ts
-import { ChainId } from "@looksrare/sdk-v2";
-const looksrare = new LooksRare(ChainId.MAINNET, provider);
-```
+The `HypercertExchangeClient()` allows you to override the API endpoint. This can be useful for developing against a local instance of the API.
 
-If you work on a hardhat setup, you can override the addresses as follow:
+To use a locally built version of the SDK in another project, you can use `pnpm link`:
 
-```ts
-import { Addresses } from "@looksrare/sdk-v2";
-const addresses: Addresses = {...};
-const looksrare = new LooksRare(ChainId.HARDHAT, provider, signer, addresses);
+```bash
+cd marketplace-sdk
+pnpm install && pnpm run build
+cd ../your-project
+pnpm link @hypercerts-org/marketplace-sdk
 ```
 
-:information_source: Use [`JsonRpcProvider`](https://docs.ethers.org/v6/api/providers/jsonrpc/#JsonRpcApiProviderOptions) from `ethers v6` if you want to make batched RPC calls.
-
-```ts
-import { JsonRpcProvider, Network } from "ethers";
-
-// Create a HTTP/HTTPS batch call provider
-const provider = new JsonRpcProvider(RPC_URL, CHAIN_ID, { staticNetwork: Network.from(CHAIN_ID) });
-
-// Create LooksRare instance using this provider
-const looksrare = new LooksRare(ChainId.HARDHAT, provider, signer, addresses);
+### Scripts
+
+- `dev` - Run the SDK in development mode
+- `build` - Build the SDK
+- `test` - Run the tests
+- `docs` - Generate the documentation into the `doc` folder
+- `supabase:types:hypercerts` - Generate types for the `data-staging` database
```

Comment:
Due to my late review these are now outdated? Because we connect to the API, not Supabase?

Reply:
Yes and no. We still use the generated types in 1 or 2 places, but never connect to supabase anymore once https://github.com/hypercerts-org/marketplace-sdk/pull/27 gets merged. It could be removed if we pull out the 1 or 2 types that we're still using but I think that belongs in the other PR.

Reply:
https://github.com/hypercerts-org/marketplace-sdk/issues/29